api/RTCPeerConnectionIceEvent.json

```diff
@@ -0,0 +1,203 @@
+{
+  "api": {
+    "RTCPeerConnectionIceEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnectionIceEvent",
+        "support": {
+          "webview_android": [
+            {
+              "alternative_name": "RTCIceCandidateEvent",
+              "version_added": true
```

Comment:
Also add:
"version_removed": "56"

Reply:
Ahh yeah, of course! :slightly_smiling_face: